panel/views/pages/tree.php

```diff
@@ -53,6 +53,18 @@
                         <div class="page-route truncate mr-2" aria-hidden="true">
                             <span><?= $page->canonicalRoute() ?? $page->route() ?></span>
                         </div>
+                        <?php
+                            $imagePreviewFieldName = $page->scheme()->options()->get('imagePreviewField', '');
+                            $imagePreviewFieldValue = $imagePreviewFieldName == '' ? '' : $page->fields()->get($imagePreviewFieldName);
+                            $imagePreviewFile = $imagePreviewFieldValue == '' ? '' : $page->files()->get($imagePreviewFieldValue);
+                        ?>
+                        <?php if($imagePreviewFile != '') : ?>
```

Comment:
We can check instead if there is a non-`null` `$imagePreviewField` of type `image`:
```suggestion
                        <?php if ($imagePreviewField !== null && $page->fields()->get($imagePreviewField)?->type() === 'image'): ?>
```